webapp/package.json

```
@@ -3,7 +3,9 @@
   "version": "0.0.1",
   "description": "This plugin serves as a starting point for writing a Mattermost plugin.",
   "main": "src/index.js",
+  "private": true,
```

We can set the version like below, just so it's defined by the other dependencies. We could move this down further in the file though, near the other dependencies. Note that `localPackages` is an arbitrary name I'm suggesting here.

```suggestion
  "private": true,
  "localPackages": {
    "mattermost-webapp": "e2a5293e2e1c99ba66c4ebd0e358a1daaad2c4ed"
  },
```